src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/CaseNoteController.kt

```diff
@@ -1,5 +1,6 @@
 package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.controller
 
+import jakarta.persistence.EntityNotFoundException
```

Comment:
So are we moving from Javax persistance to Jakara persistence

Reply:
Yes we have to move over javax isn't there any more